.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,19 @@
+mkdir -p $HOME/.kube
+kubectl completion bash > /home/vscode/.kube/completion.bash.inc
+printf "
+source /usr/share/bash-completion/bash_completion
+source $HOME/.kube/completion.bash.inc
+alias k=kubectl
```

Comment:
small request from my side:
```
alias ka='kubectl --as cluster-admin'
alias kwp='watch -n 0.1 kubectl get pod'
alias kwo='watch -n 0.1 kubectl get object'
alias kwoa='watch -n 0.1 kubectl --as cluster-admin get object'
```

Reply:
alias of `ll` is also missing in bash spec

Reply:
You can add your own aliases via dotfiles. I wouldn't clutter the defaults too much, as everybody wants different things.

Here's an example of my dotfiles: https://git.earthnet.ch/simon.beck/devcontainerdotfiles/src/branch/main/zshrc